.gitignore

```diff
@@ -13,3 +13,44 @@ Shaders/Compiled/*spv
 Shaders/bintoc.exe
 Shaders/bintoc.obj
 Windows/VisualStudio/.vs
+
+.DS_Store
+
+
```

Comment:
The ignoring and removal of these generated files is very rude. One would prefer them to be present in the source tree.

Reply:
I have made a fix for this, in the next commit. Since shaders are platform independent (and gpu interchangeable) it make sense to provide pre-compiled shaders to endusers.

Though, I'm leaving the binary "bintoc" in the .gitignore ... since it's not platform interchangeable .. and the compile.sh scripts handles it, if it's missing by compiling one for the users platform. 

Also, it would be best if each user from time to time re-compiled them on their own ... as the Vulkan SDK advances and better and maybe more efficient compiled shaders can be produces. I will add this into a build script, to re-compile them, and then the source code.

Reply:
You have a misconception there. I don't use glslang to optimize. The SPIR-V is a pretty literal translation from GLSL. Optimization should be done by the driver. The glslang optimization options just exist because of absurdly garbage mobile drivers and are counter productive on PC.

Reply:
The generated `*.c` are still removed in the latest version of your patch.

Reply:
> The generated `*.c` are still removed in the latest version of your patch.

They should be, please see: https://github.com/Novum/vkQuake/pull/266/commits/149c6df7f56eab2ecfa9182c47c0aa77140da2e1

Reply:
> You have a misconception there. I don't use glslang to optimize. The SPIR-V is a pretty literal translation from GLSL. Optimization should be done by the driver. The glslang optimization options just exist because of absurdly garbage mobile drivers and are counter productive on PC.

👍🏻 ... thanks for clarifying it 🙂 *happy-to-learn*

Though, I'm wondering a bit how Apple will treat shaders with their new M1, and future to come gpu's ... since they do have a "if you can, please optimise your code to make it fly on our hardware" ... I don't know, but felt it's better to make a build script that re-compiles them before each build (in case, in the future, things change... and re-compile is beneficiary...)

Reply:
It's not beneficiary. Drop it.

Reply:
Honsestly, I don't know how this PR is beneficial in whole: https://github.com/Novum/vkQuake/pull/266#issuecomment-740045611